services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AbstractDeviceManagementService.java

```diff
@@ -225,6 +225,7 @@ protected Future<Result<Void>> processDeleteDevicesOfTenant(final String tenantI
      *                   retrieve the whole result set page by page.
      * @param filters A list of filters. The filters are predicates that objects in the result set must match.
      * @param sortOptions A list of sort options. The sortOptions specify properties to sort the result set by.
+     * @param isGateway Optional filter for searching all gateways or only devices.
```

Comment:
can you please make it more explicit what will be returned if the given parameter is `Optional.empty()`?

Reply:
Done